core/common/src/serializers/MonthSerializers.kt

```diff
@@ -12,7 +12,7 @@ import kotlinx.serialization.encoding.*
 import kotlinx.serialization.internal.*
 
 /**
- * A serializer for [Month] that represents the values as strings.
+ * A serializer for [Month] that represents the month value as a string.
```

Comment:
It is correct, but I don't see the need for redundancy. What other values could the serializer operate on? So, I'd suggest either "a value", or "the values", which I still prefer.